GliaWidgets/SecureConversations/Confirmation/SecureConversations.ConfirmationView.swift

```diff
@@ -0,0 +1,171 @@
+import Foundation
+import UIKit
+
+extension SecureConversations {
+    final class ConfirmationView: View {
+        static let sideMargin = 24.0
+        static let confirmationImageSize = 144.0
+
+        struct Props: Equatable {
+            let style: ConfirmationStyle
+            let backButtonTap: Cmd
+            let closeButtonTap: Cmd
+            let checkMessageButtonTap: Cmd
+        }
+
+        private let header: Header
+
+        var topRootStackViewConstraint: NSLayoutConstraint?
+        lazy var rootStackView = UIStackView().makeView { stackView in
+            stackView.axis = .vertical
+            stackView.alignment = .center
+            stackView.distribution = .fillProportionally
+        }
```

Comment:
perhaps it could be simplified like this and added arranged view here also. In this case, the order of arranged views will be more clear.
```swift
// Flexible space to accommodate the check messages button
// at the bottom of the view.
let spacer = UIView()
lazy var rootStackView =UIStackView.make(.horizontal, spacing: 8, distribution: .fillProportionally)(
    confirmationImageView,
    titleLabel,
    spacer,
    checkMessagesButton
)

```

Reply:
Like this? c98fb67a3ac80d75cb9a5ff763d82a4b062aaafe